src/Root.cpp

```diff
@@ -141,8 +141,14 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 
 	LOG("Reading server config...");
 
+	std::string conf_file = "settings.ini";
+	if (a_OverridesRepo->HasValue("Server","ConfigFile"))
+	{
+	conf_file = a_OverridesRepo->GetValue("Server","ConfigFile");
```

Comment:
You should add an extra tab for `conf_file = a_OverridesRepo..`

With the brace you're opening a new block, so by adding a new tab you're making the piece of code much more readable. This is especially more noticable if there is more code inside a block,